File: backend/host/Models/OrderItem.cs
using System.Text.Json.Serialization;

namespace host.Models;

public class OrderItem
{
    public int? Id { get; set; } 
    [JsonPropertyName("order_id")] public int OrderId { get; set; }
    public int Count { get; set; }
    public double Price { get; set; }
    [JsonPropertyName("menu_item")] public MenuItem MenuItem { get; set; }
    
    public OrderItem(int? id, int orderId, int count, double price, MenuItem menuItem)
    {
        Id = id;
        OrderId = orderId;
        Count = count;
        Price = price;
        MenuItem = menuItem;
    }
}

File: frontend/Business/tg_business_bot/Models/OrderItem.py
from pydantic import BaseModel
from typing import Optional

from Models.MenuItem import MenuItem


class OrderItem(BaseModel):
    id: Optional[int]
    order_id: int
    menu_item: MenuItem
    count: int
    price: float

    def __init__(self,
                 order_id: int,
                 menu_item: MenuItem,
                 count: int,
                 price: float,
                 id: Optional[int] = None):
        super().__init__(
            id=id,
            order_id=order_id,
            menu_item=menu_item,
            count=count,
            price=price
        )
